setup/cluster/README.md

```diff
@@ -0,0 +1,228 @@
+# Connaisseur with existing Kubernetes cluster
+
+This guide will show you how to set up Connaisseur on your Kubernetes cluster. Unfortunately, most cloud vendors' container registries do not support Docker Content Trust (DCT) at this point in time. Hence, we will not be using them and, also for simplicity, instead go for Docker Hub, which combines both a registry and a notary server. If you want to setup your own registry and notary, e.g. Harbor, checkout the [Setup Guide](../local/README.md), which shows how to work with a local instance of Harbor.
+
+## Requirements
+
+We assume you have a running Kubernetes cluster already. Since this tutorial works with the CLI, we furthermore assume you have the `docker`, `git`, `helm`, `kubectl`, `make` binaries installed and usable, i.e. having run `docker login` and having switched to the appropriate `kubectl` context.
+
+> This tutorial was tested on a machine running Ubuntu 20.04. It worked an Azure Kubernetes Service (AKS) cluster, an Elastic Kubernetes Service (EKS) cluster based on EC2 instances and with a local minikube cluster.
+
+## Starting and testing a deployment of Connaisseur
+
+### 1. Set up environment
+
+First off, we'll export the variables used in this tutorial to allow you to immediately use your own names/URLs instead of relying on our default names. Below, substitute your own values as appropriate:
+
+```bash
+REGISTRY_URL=docker.io
+REPOSITORY_NAME=testingconny
+IMAGE_NAME=testimage
+IMAGE_PATH=$(echo ${REGISTRY_URL}/${REPOSITORY_NAME}/${IMAGE_NAME})
```

Comment:
I think you only ever use the `IMAGE_PATH` and thus I would only set that. Otherwise, it makes it seem overly complicated. Maybe make a comment to show a sample entry.